CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Mapbox welcomes participation and contributions from everyone.
 * Optimise the frequency to update location layer's visibility. ([1399](https://github.com/mapbox/mapbox-maps-android/pull/1399))
 * Fix the delay in the first appearance of the location puck. ([1403](https://github.com/mapbox/mapbox-maps-android/pull/1403))
 
+## Dependencies
+* Bump telemetry to 8.1.3, android core to 5.0.2. ([1430](https://github.com/mapbox/mapbox-maps-android/pull/1430))
```

Comment:
if we're moving that to 10.6.0 for sure - guess it's easier to put it under 10.6.0 section otherwise we could forget to leave it here and we will state that bump for 10.7.0-beta.1

Reply:
also I assume if that's fix is critical - would be nice to state what exactly changed in changelog

Reply:
Updated changelog and added the permission removal of `android.permission.GET_TASKS`